schema/revert/tables/emission_category.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ggircs-portal:tables/emission_category from pg
+
+BEGIN;
```

Comment:
```suggestion
begin;
```
(just for consistency with other files)